src/Shared/Components/ButtonWIthLoader/ButtonWithLoader.tsx

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable react/destructuring-assignment */
+import React, { Component } from 'react'
+import { Progressing } from '../../../Common'
+
+export interface ButtonProps {
+    disabled?: boolean
+    rootClassName: string
+    isLoading: boolean
+    onClick?: (event) => void
+    dataTestId?: string
+    children?: React.ReactNode
+    type?: 'submit' | 'reset' | 'button'
+}
+
+export class ButtonWithLoader extends Component<ButtonProps> {
```

Comment:
We have not exported anywhere, pls verify the usage after exporting for one case